src/Quartz/Core/ListenerManagerImpl.cs

```diff
@@ -12,14 +12,12 @@ namespace Quartz.Core
     public class ListenerManagerImpl : IListenerManager
     {
         private readonly object globalJobListenerLock = new object();
-
         private OrderedDictionary<string, IJobListener>? globalJobListeners;
-
-        private readonly OrderedDictionary<string, ITriggerListener> globalTriggerListeners = new OrderedDictionary<string, ITriggerListener>(10);
-
         private Dictionary<string, List<IMatcher<JobKey>>>? globalJobListenersMatchers;
 
-        private readonly Dictionary<string, List<IMatcher<TriggerKey>>> globalTriggerListenersMatchers = new Dictionary<string, List<IMatcher<TriggerKey>>>(10);
+        private readonly object globalTriggerListenerLock = new object();
+        private OrderedDictionary<string, ITriggerListener>? globalTriggerListeners;
```

Comment:
This is quite the micro-optimization for applications that usually have single scheduler